.circleci/config.yml

```diff
@@ -1,977 +1,67 @@
----
 version: 2.1
 
+setup: true
+
 orbs:
-  gcp-gcr: circleci/gcp-gcr@0.13.0
-  docker: circleci/docker@2.2
-  python: circleci/python@2.1.1
+  path-filtering: circleci/path-filtering@0.1.3
 
 parameters:
```

Comment:
Some of them are used in the path-filter mapping. But I'll remove the ones not being used.

Reply:
From what I could tell looking at the path-filtering orb code it doesn't seem to do anything with parameters in the config it's used in, it's only defining parameters to pass to a downstream config.  So the parameters are definitely needed in `workflows.yml`, but I don't think they're needed here.

Reply:
Ah right, they are not needed for path-filtering. However they need to be listed to allow manual triggering of specific workflows. Otherwise there is an unexpected argument error: https://app.circleci.com/pipelines/gh/mozilla/bigquery-etl/30607

Reply:
OK.  Does there need to be extra logic to pass through the parameter values to `workflows.yml` if it's triggered manually?

Reply:
No extra logic needed. It just passes them through.